### DISCUSSION TITLE
ФРМО - Вид деятельности, профиль деятельности – одно и то же?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#66] [Административная РГ]
Строки 160, 161 вид деятельности. Заполняется по справочнику 1.2.643.5.1.13.13.99.2.289
По тому же справочнику заполняется поле профиль деятельности - строки 162, 163.

Это иерархический справочник, где на верхнем уровне типовые названия:
 
<img width="146" alt="image" src="https://user-images.githubusercontent.com/62525329/146547885-e5098b5e-d69c-4f2f-90d4-44d4c0a65700.png">

На втором уровне некоторые позиции разложены по медицинским профилям (избранным):
<img width="271" alt="image" src="https://user-images.githubusercontent.com/62525329/146547912-f1b0cab7-7f43-4578-837a-b81bc144d9d9.png">
 
Иногда на втором уровне приведены типовые варианты организаций, для которых первый уровень просто объединяет по части названия (центр, лаборатория):
<img width="452" alt="image" src="https://user-images.githubusercontent.com/62525329/146547944-847bdfc1-dece-4123-bdfa-0cef8db5b613.png">

Вопрос: в чем смысл информации.

**Comment by IvanekOS** [comment]
> Строки 160, 161 вид деятельности. Заполняется по справочнику 1.2.643.5.1.13.13.99.2.289 По тому же справочнику заполняется поле профиль деятельности - строки 162, 163.

Согласен, с тем, что текущий формат, включающий в себя множественное кодирование связанной между собой информации, по одному справочнику является бессмысленным. Вполне достаточно использовать одну запись.

Справочник 1.2.643.5.1.13.13.99.2.289 формировался на основе приказа МЗ РФ 06.08.2013 г. № 529н (Раздел I). В соответствии с текущим справочником ФРМО, размещенном на НСИ, к видам медицинской помощи относится верхний уровень, а к профилям  относится второй уровень.

Причина добавления этой информации скорее всего аналогична полю "Территориальный признак": добавлено в соответствии с приказом МЗ РФ 06.08.2013 г. № 529н.

**Reply by Vrodvi** [reply]
Данный комментарий не отвечает на поставленный вопрос - "ФРМО - Вид деятельности, профиль деятельности – одно и то же?"

Судя по указанному приказу (МЗ РФ 06.08.2013 г. № 529н (Раздел I)) определена одна одна номенклатура -
"I. Номенклатура медицинских организаций <*> по виду медицинской деятельности"
, в приказе нет деления на вид и профиль.
**Поэтому просьба всё же уточнить на основании чего и для каких целей было введено данное деление?**

Похоже, что должно быть просто отнесение организации к наиболее детализированному варианту из номенклатуры, например:
"- Лечебно-профилактические медицинские организации:
	-- 1.4.Специализированные больницы (в том числе по профилю медицинской помощи), а также специализированные больницы государственной и муниципальной систем здравоохранения:
		--- гинекологическая;"

При этом, возможно что указание основного "профиля медицинской помощи" для организации также необходимо, но тогда его нужно кодировать по соответствующему стандартному справочнику - 
||||
| --- | --- | --- |
| [1.2.643.5.1.13.13.11.1119](https://nsi.rosminzdrav.ru/#!/refbook/1.2.643.5.1.13.13.11.1119/passport/2.2) | Профили медицинской помощи | Основные объекты НСИ |
